libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -321,7 +324,8 @@ void elasticsearch_plugin_impl::prepareBulk(const account_transaction_history_id
    bulk_header["_type"] = "data";
    bulk_header["_id"] = fc::to_string(ath_id.space_id) + "." + fc::to_string(ath_id.type_id) + "." + ath_id.instance;
    prepare = graphene::utilities::createBulk(bulk_header, bulk_line);
```

Comment:
can use std::move(bulk_line) here if createBulk method signature is adjusted accordingly